data/accounts/fr_CH/acctchrt_pme-19.gnucash-xea

```diff
@@ -0,0 +1,2132 @@
+    <act:commodity>
+    <cmdty:space>CURRENCY</cmdty:space>
+    <cmdty:id>CHF</cmdty:id>
+    </act:commodity>
+    <act:commodity-scu>100</act:commodity-scu>
+    <act:code>8100</act:code>
+    <act:description>8100</act:description>
+    <act:parent type="guid">67be6ac625cc4b8f8ee6706b0434a335</act:parent>
+  </gnc:account>
+  <gnc:account version="2.0.0">
+    <act:name>Charges extraordinaires, exceptionnelles ou hors période</act:name>
+    <act:id type="guid">e3b24b1150134fa598bcf620da52a2e6</act:id>
+    <act:type>INCOME</act:type>
+    <act:commodity>
+    <cmdty:space>CURRENCY</cmdty:space>
+    <cmdty:id>CHF</cmdty:id>
+    </act:commodity>
+    <act:commodity-scu>100</act:commodity-scu>
+    <act:code>8500</act:code>
+    <act:description>8500</act:description>
+    <act:parent type="guid">67be6ac625cc4b8f8ee6706b0434a335</act:parent>
+  </gnc:account>
+  <gnc:account version="2.0.0">
+    <act:name>Produits extraordinaires, exceptionnels ou hors période</act:name>
+    <act:id type="guid">7e43a5e29f77460dbda26d475492bd52</act:id>
+    <act:type>INCOME</act:type>
+    <act:commodity>
+    <cmdty:space>CURRENCY</cmdty:space>
+    <cmdty:id>CHF</cmdty:id>
+    </act:commodity>
+    <act:commodity-scu>100</act:commodity-scu>
+    <act:code>8510</act:code>
+    <act:description>8510</act:description>
+    <act:parent type="guid">67be6ac625cc4b8f8ee6706b0434a335</act:parent>
+  </gnc:account>
+  <gnc:account version="2.0.0">
+    <act:name>Clôture</act:name>
+    <act:id type="guid">99c8d4fa6bac4de0b2d505f65e9b8c6b</act:id>
+    <act:type>INCOME</act:type>
+    <act:commodity>
+    <cmdty:space>CURRENCY</cmdty:space>
+    <cmdty:id>CHF</cmdty:id>
+    </act:commodity>
+    <act:commodity-scu>100</act:commodity-scu>
+    <act:code>9</act:code>
+    <act:description>9</act:description>
+    <act:parent type="guid">8cd23636f1e04a74a35c13ae23427c14</act:parent>
+  </gnc:account>
+  <gnc:account version="2.0.0">
+    <act:name>Bénéfice / perte de l’exercice</act:name>
+    <act:id type="guid">3b10783f477e440b971e3dce693236e5</act:id>
+    <act:type>INCOME</act:type>
+    <act:commodity>
+    <cmdty:space>CURRENCY</cmdty:space>
+    <cmdty:id>CHF</cmdty:id>
+    </act:commodity>
+    <act:commodity-scu>100</act:commodity-scu>
+    <act:code>9200</act:code>
+    <act:description>9200</act:description>
+    <act:parent type="guid">99c8d4fa6bac4de0b2d505f65e9b8c6b</act:parent>
+  </gnc:account>
+  <gnc:account version="2.0.0">
+    <act:name>Comptes auxiliaires d'affectation du bénéfice</act:name>
+    <act:id type="guid">be48f890d33444f883f41069641daaa3</act:id>
+    <act:type>INCOME</act:type>
+    <act:commodity>
+    <cmdty:space>CURRENCY</cmdty:space>
+    <cmdty:id>CHF</cmdty:id>
+    </act:commodity>
+    <act:commodity-scu>100</act:commodity-scu>
+    <act:code>920</act:code>
+    <act:description>920</act:description>
+    <act:parent type="guid">3b10783f477e440b971e3dce693236e5</act:parent>
+  </gnc:account>
```

Comment:
You should add 2 additionally accounts:
Equity:Opening Balance
the last needs additionally in slots:
```
     <slot>
      <slot:key>equity-type</slot:key>
      <slot:value type="string">opening-balance</slot:value>
    </slot>
```
to mark it as target for opening transactions.

Reply:
Thanks for the feedback, I'm currently working on it + improving the translation/structure based on the following official document:  https://weiterbildung.veb.ch/hubfs/01_Content/03_Dokumente/02_Schulkontorahmen/Schulkontenrahmen_veb.ch_franz.pdf

I'm still working on the last comment about the opening balance, then I'll do a full review and commit the changes.